sdk/eventgrid/Azure.Messaging.EventGrid/src/EventGridEvent.cs

```diff
@@ -1,10 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#nullable enable
```

Comment:
We aren't enabling nullable references in any of our public APIs in the Azure SDK as we haven't fully fleshed out the versioning story.

Reply:
I should have asked. That leaves basically the if check to avoid the null ref on ToMemory and the cosmetics. Is there any value in those then I'm happy to reset parts of the changes and retitle the PR?

Reply:
No worries 😄 Sure, that sounds good.